src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,13 @@
+import { h, Component } from "preact";
+import * as styles from "./spinner.scss";
+
+export class Spinner extends Component {
```

Comment:
please coordinate with Shai / Adi. If this is the official Kaltura logo you need to move it to contrib-ui and refactor both plugins